sympy/printing/pretty/pretty_symbology.py

```diff
@@ -449,21 +449,22 @@ def xsym(sym):
 # SYMBOLS
 
 atoms_table = {
-    # class             how-to-display
-    'Exp1':             U('SCRIPT SMALL E'),
-    'Pi':               U('GREEK SMALL LETTER PI'),
-    'Infinity':         U('INFINITY'),
-    'NegativeInfinity': U('INFINITY') and ('-' + U('INFINITY')),  # XXX what to do here
-    #'ImaginaryUnit':    U('GREEK SMALL LETTER IOTA'),
-    #'ImaginaryUnit':    U('MATHEMATICAL ITALIC SMALL I'),
-    'ImaginaryUnit':    U('DOUBLE-STRUCK ITALIC SMALL I'),
-    'EmptySet':         U('EMPTY SET'),
-    'Naturals':         U('DOUBLE-STRUCK CAPITAL N'),
-    'Integers':         U('DOUBLE-STRUCK CAPITAL Z'),
-    'Reals':            U('DOUBLE-STRUCK CAPITAL R'),
-    'Union':            U('UNION'),
-    'Intersection':     U('INTERSECTION'),
-    'Ring':             U('RING OPERATOR')
+    # class                    how-to-display
+    'Exp1':                    U('SCRIPT SMALL E'),
```

Comment:
@hargup `atoms_table` is a dict for accessing unicode symbols from the `unicodedata` module. Here the key `SymmetricDifference` is used in `pretty.py` module to print the `INCREMENT` unicode symbol for `SymmetricDifference` class. 
In case of `Complement` class you created, there was no need for any `unicode` symbol as `\` was used for `unicode` and `ascii` printing, which can be directly used.